Yoast/YoastSEO.js
# 1809 load morphology data

## Summary

This PR can be summarized in the following changelog entry:

* YoastSEO.js side of loading and injecting data for morphology analysis from MyYoast, through Premium.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo-premium/pull/2038 for test instructions.

Fixes #1809
